ClimsoftVer4/ClimsoftVer4/ucrTextBox.vb

```diff
@@ -276,38 +296,55 @@ Public Class ucrTextBox
         OnevtTextChanged(sender, e)
 
     End Sub
-
+    ''' <summary>
+    ''' Sets the focus to the control 
+    ''' </summary>
     Public Sub GetFocus()
         txtBox.Focus()
     End Sub
-
+    ''' <summary>
+    ''' Checks if a textbox is empty
+    ''' Returns true when text box is empty
+    ''' </summary>
+    ''' <returns></returns>
     Public Function IsEmpty() As Boolean
         If TextboxValue.Length > 0 Then
             Return False
         Else
             Return True
         End If
     End Function
-
+    ''' <summary>
+    ''' Clears contents of the textbox
+    ''' </summary>
     Public Overrides Sub Clear()
         bValidate = False
         TextboxValue = ""
```

Comment:
@dannyparsons was of the opinion that in a situation like this, instead of TextboxValue = "" we do setValue(""). Since its the new way of setting values for all custom controls. Probably you can change this also if its necessary